db/migrate/20181026182144_add_tasks_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddTasksTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tasks, :id => :bigserial do |t|
+      t.string :name
+      t.string :status
+      t.string :state
+      t.string :message
+      t.integer :progress
+      t.references :tenant, :index => true, :null => false
+      t.timestamps
```

Comment:
Should we also have a `completed_on` timestamp?

Reply:
Yeah good idea